webpack/components/extensions/HostDetails/Tabs/RepositorySetsTab/RepositorySetsTab.js

```diff
@@ -66,14 +66,15 @@ const RepositorySetsTab = () => {
     content_facet_attributes: contentFacetAttributes,
   } = hostDetails;
   const contentFacet = propsToCamelCase(contentFacetAttributes ?? {});
-  const nonLibraryHost = !contentFacet.contentViewDefault &&
-  !contentFacet.lifecycleEnvironmentLibrary;
+  const nonLibraryHost = contentFacet.contentViewDefault === false &&
+    contentFacet.lifecycleEnvironmentLibrary === false;
```

Comment:
When the page is loading, these two values were `undefined` which was causing `alertShowing`'s default state to be incorrect. The result was that the alert would show even on a Library host if you reloaded the page.  This fixes that.